capa/ida/plugin/view.py

```diff
@@ -194,7 +194,9 @@ def load_preview_meta(self, ea, author, scope):
             "    namespace: <insert_namespace>",
             "    authors:",
             f"      - {author}",
-            f"    scope: {scope}",
+            "    scopes:",
+            f"      static: {scope}",
+            "      dynamic: unspecified",
```

Comment:
```suggestion
            "    scopes:",
            f"      static: {scope}",
            "       dynamic: unspecified",
```

Reply:
note the f-string indent above, so this is correct as is